src/twisted/internet/base.py

```diff
@@ -1144,7 +1144,7 @@ def _initThreads(self) -> None:
             self.installNameResolver(_GAIResolver(self, self.getThreadPool))
             self.usingThreads = True
 
-        def callFromThread(
+        def callFromThread(  # type: ignore[override]
             self, f: Callable[..., Any], *args: object, **kwargs: object
         ) -> None:
```

Comment:
the trouble is `twisted.internet.interfaces.IReactorFromThreads` defines the first named argument as `callable: Callable[..., Any]`

Reply:
Can you move this explanation to a comment?
We should be documenting ignores that aren't obvious.